weblate/trans/models/component.py

```diff
@@ -1176,6 +1207,118 @@ def get_repoweb_link(
             component=self,
         )
 
+    def get_git_repoweb_template(self):
+        if any(regex.match(self.repo) for regex in BITBUCKET_GIT_REPOS_REGEXP):
```

Comment:
Aren't these expressions matching nearly anything? I'd rather do guess based on the domain name only.

Reply:
1. Can you please add different example cases for bitbucket, pagure and azure to make the guess on domain only.

2. Something like this `git@github.com:([^/]*)\/([^/]*)` is expected here after domain only guess, or just like `github.com` in `str` instance is enough?

Reply:
Maybe something like this:

```python
parsed_url = urlparse(self.repo)
if self.repo.startswith("git@github.com") or parsed_url.hostname == "github.com":
    ...
```